projects/packages/stats-admin/tests/php/test-odyssey-config-data.php

```diff
@@ -42,6 +42,6 @@ public function test_config_data() {
 		$this->assertArrayHasKey( 'nonce', $data );
 		$this->assertArrayHasKey( 'site_name', $data );
 		$this->assertArrayHasKey( 'intial_state', $data );
-		$this->assertEmpty( $data['features'] );
+		$this->assertNotEmpty( $data['features'] );
```

Comment:
This should get the tests passing, but maybe it should be more specific? 🤔

Reply:
Made it more specific 👍